packs/kingmaker-bestiary/cursed-guardian.json

```diff
@@ -109,7 +109,25 @@
                     "remaster": false,
                     "title": ""
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "definition": [
+                            "deity:primary:erastil"
+                        ],
```

Comment:
Does this one not need the origin?

Reply:
I tested `"origin:deity:primary:erastil"`, and it doesn't work. However, the predicate I used for the FlatModifer `"origin:deity:slug:erastil"` does. I switched to that predicate so they'd be the same.